src/test/java/com/butbetter/applicationServices/CalculatorTest_mockito.java

```diff
@@ -0,0 +1,72 @@
+package com.butbetter.applicationServices;
+
+import com.butbetter.applicationServices.model.CalcApiResponse;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.web.client.HttpClientErrorException;
+import org.springframework.web.client.ResourceAccessException;
+import org.springframework.web.client.RestTemplate;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+@SpringBootTest
```

Comment:
I think you can use @AutoConfigureMockMvc

Reply:
i will try that looks like it could fix my issues with mocking the rest API 👍